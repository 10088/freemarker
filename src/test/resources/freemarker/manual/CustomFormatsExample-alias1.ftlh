${p?string.@price}
${w?string.@weight}
${fd?string.@fileDate}
${let?datetime?string.@logEventTime}
