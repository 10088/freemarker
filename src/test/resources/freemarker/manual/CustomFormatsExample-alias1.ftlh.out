10,000.00
10.31
23/Dec/15 10:09 PM
2015-12-23T21:09:04.213Z
